### ISSUE TITLE
delonghi  primadonna ECAM550.65.W ?

### ISSUE THREAD

**Issue by simona70** [Arbuzov/home_assistant_delonghi_primadonna#104]
delonghi  primadonna ECAM550.65.W ?

**Comment by GSzabados** [comment]
Yes, it is working with it. I am submitting a few PRs to make it more accurate.

Could you get some Bluetooth DEBUGS logs? Just to see that if is there something which defines it clearly that it is a Primadonna Class?
